cluster/terraform_kubernetes/welcome_app.tf

```diff
@@ -114,3 +167,29 @@ resource "kubernetes_ingress_v1" "welcome_app" {
     }
   }
 }
+
+resource "kubernetes_config_map" "nginx_config" {
+  metadata {
+    name      = "nginx-config"
```

Comment:
Can we rename to welco-app-nginx-config ?

Reply:
fixed